### ISSUE TITLE
how to train own datasets

### ISSUE THREAD

**Issue by xingjunhong** [zhengchen1999/DAT#15]
how to train own datasets

**Comment by zhengchen1999** [comment]
You can modify the training configs in `.yml` files, like [train_DAT_light_x4.yml](https://github.com/zhengchen1999/DAT/blob/main/options/Train/train_DAT_light_x4.yml#L14).
Replace your own data (LR and HR) path in `dataroot_gt` and `dataroot_lq`.
```shell
    dataroot_gt: datasets/DF2K/HR
    dataroot_lq: datasets/DF2K/LR_bicubic/X4
```
PS: Our code is based on [BasicSR](https://github.com/XPixelGroup/BasicSR). It provide detail commands tutorials in [doc](https://github.com/XPixelGroup/BasicSR/tree/master/docs). Maybe you refer to it.

**Comment by xingjunhong** [comment]
How much data does it take to train a demo model

**Comment by zhengchen1999** [comment]
The DK2K contains 3550 images.
The training setting of our models: batch size = 32, patch size = 64 $\times$ 64, training iteration = 500,000.

**Comment by xingjunhong** [comment]
Are we training with high-definition images? Or use low resolution images for training?

**Comment by zhengchen1999** [comment]
Training needs high-resolution and low-resolution image pairs.
This is a supervised task, where high-resolution images are ground truth.

**Comment by xingjunhong** [comment]
Isn't it using low resolution images for training and then performing loss calculations with high-resolution images?

**Comment by zhengchen1999** [comment]
Yes.

**Comment by xingjunhong** [comment]
Is the original image of the dataset we collected high-resolution? Then use the script to turn it into a low resolution image and train it with a low resolution image?

**Comment by zhengchen1999** [comment]
Yes, in our approach, we use bicubic downsampling for high-resolution images (x2, x3, x4). Of course, you can also utilize other methods, such as the degradation model in [Real-ESRGAN](https://github.com/xinntao/Real-ESRGAN), which is suitable for real-world SR.
